code/modules/pool/pool_main.dm

```diff
@@ -0,0 +1,278 @@
+
+/turf/open/pool/Entered(atom/A, turf/OL)
+	..()
+	if(isliving(A))
+		var/mob/living/M = A
+		if(!M.mob_has_gravity())
+			return
+		if(!M.swimming)
+			M.swimming = TRUE
+			controller.mobs_in_pool.Add(M)
+			if(locate(/obj/structure/pool/ladder) in M.loc)
+				return
+			if(iscarbon(M))
+				var/mob/living/carbon/H = M
+				if(filled)
+					wash_mob(H)
+					if (H.wear_mask && H.wear_mask.flags_cover & MASKCOVERSMOUTH)
+						H.visible_message("<span class='danger'>[H] falls in the water!</span>",
+											"<span class='userdanger'>You fall in the water!</span>")
+						playsound(src, 'sound/effects/splash.ogg', 60, TRUE, 1)
+						H.Knockdown(20)
+						return
+					else
+						H.dropItemToGround(H.get_active_held_item())
+						H.adjustOxyLoss(5)
+						H.emote("cough")
+						H.visible_message("<span class='danger'>[H] falls in and takes a drink!</span>",
+											"<span class='userdanger'>You fall in and swallow some water!</span>")
+						playsound(src, 'sound/effects/splash.ogg', 60, TRUE, 1)
+						H.Knockdown(60)
+				else if(!istype(H.head, /obj/item/clothing/head/helmet))
+					if(prob(75))
+						H.visible_message("<span class='danger'>[H] falls in the drained pool!</span>",
+													"<span class='userdanger'>You fall in the drained pool!</span>")
+						H.adjustBruteLoss(7)
+						H.Knockdown(80)
+						playsound(src, 'sound/effects/woodhit.ogg', 60, TRUE, 1)
+					else
+						H.visible_message("<span class='danger'>[H] falls in the drained pool, and cracks his skull!</span>",
+													"<span class='userdanger'>You fall in the drained pool, and crack your skull!</span>")
+						H.apply_damage(15, BRUTE, "head")
+						H.Knockdown(200) // This should hurt. And it does.
+						H.adjustBrainLoss(30) //herp
+						playsound(src, 'sound/effects/woodhit.ogg', 60, TRUE, 1)
+						playsound(src, 'sound/misc/crack.ogg', 100, TRUE)
+				else
+					H.visible_message("<span class='danger'>[H] falls in the drained pool, but had an helmet!</span>",
+										"<span class='userdanger'>You fall in the drained pool, but you had an helmet!</span>")
+					H.Knockdown(40)
+					playsound(src, 'sound/effects/woodhit.ogg', 60, TRUE, 1)
+		else if(filled)
+			wash_mob(M)
+			M.adjustStaminaLoss(1)
+			playsound(src, "water_wade", 20, 1)
+			return
+
+/turf/open/pool/attack_hand(mob/living/user)
+	if(user.stat == CONSCIOUS && !(user.lying || user.resting) && Adjacent(user) && user.swimming && filled && next_splash < world.time) //not drained, user alive and close, and user in water.
+		if(user.x == x && user.y == y)
+			return
+		else
+			playsound(src, 'sound/effects/watersplash.ogg', 8, TRUE, 1)
+			next_splash = world.time + 25
+			var/obj/effect/splash/S = new /obj/effect/splash(user.loc)
+			animate(S, alpha = 0,  time = 8)
+			S.Move(src)
+			QDEL_IN(S, 20)
+			for(var/mob/living/carbon/human/L in src)
+				if(!L.wear_mask && !user.stat) //Do not affect those underwater or dying.
+					L.emote("cough")
```

Comment:
what?

Reply:
you can splash people in the pool which causes them to cough and lose stamina.